apps/etterlatte-brev-api/.nais/dev.yaml

```diff
@@ -50,6 +50,14 @@ spec:
       value: https://saf.dev-fss-pub.nais.io
     - name: SAF_SCOPE
       value: api://dev-fss.teamdokumenthandtering.saf-q1/.default
+    - name: DOKARKIV_URL
+      value: https://dokarkiv.dev-fss-pub.nais.io/rest/journalpostapi/v1/journalpost
+    - name: DOKARKIV_SCOPE
+      value: api://dev-fss.teamdokumenthandtering.dokarkiv-q1/.default
+    - name: DOKDIST_URL
+      value: https://dokdistfordeling-q1.dev-fss-pub.nais.io/rest/v1
+    - name: DOKDIST_SCOPE
+      value: api://dev-fss.teamdokumenthandtering.saf-q1/.default
```

Comment:
Dette er samme som SAF_SCOPE så trenger kanskje ikke å ha begge?

Reply:
Hvis det er to ulike tjenester bør vi nok kunne endre dette separat. Kan være ulikt i prod også?

Reply:
Ja, jeg tenker også det er lurt å ha de adskilt.